### PR TITLE
Save knowledge

### PR DESCRIPTION
**Pull request by JosiahRegencia**
this is for issue #53 

In the database, there were some blank columns. Instead of having those blank columns I replaced them with "N/A". I tried using null, but the result was that it duplicated in the tables. Going to make a demo to show how it works.

### CODE REVIEW

**Comment by JosiahRegencia** [comment]
What's left of the issues in codeclimate are similar code issues. Which were the same ones we bypassed few meetings ago.

File: savant/app/get_scanned.py [hunk 1]
```diff
@@ -44,28 +54,35 @@ def get_debs(filename):
...
-        for data in content:
-            parts = data.rstrip('\n').split(':')
+    for data in content:
+        parts = data.rstrip('\n').split(':')
+
+        (group_name, password, gid, users) = parts[0:4]
+
+        parts[0:4] = replace_blank(parts[0:4])
+
+        # for i in xrange(len(parts)):
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: please remove commented code

File: savant/app/get_scanned.py [hunk 2]
```diff
@@ -77,12 +94,17 @@ def get_shadow(filename):
         (user_name, password, lastchanged, minimum, maximum,
                         warn, inactive, expire, reserved) = parts[0:9]
 
-        if parts[0:9] not in shadow:
-            shadow.append(parts[0:9])
-        else:
-            continue
+        parts[0:9] = replace_blank(parts[0:9])
+
+        # for i in xrange(len(parts)):
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: please remove commented code

File: savant/app/get_scanned.py [hunk 3]
```diff
@@ -94,7 +116,9 @@ def get_users(filename):
 
         (user_name, password, uid, gid, description, path, shell) = parts[0:7]
 
-        if parts[0:9] not in users:
-            users.append(parts[0:7])
-        else:
-            continue
+        parts[0:7] = replace_blank(parts[0:7])
+
+        # for i in xrange(len(parts)):
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: please remove commented code

**Review by greenmoss — CHANGES_REQUESTED** [review]

**Review by greenmoss — APPROVED** [review]

**Pull request merged**